.github/workflows/quality-assurance-php.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         if: ${{ (github.event_name != 'workflow_dispatch') || ((github.event.inputs.jobs == 'Run all') || (github.event.inputs.jobs == 'Run lint only')) }}
         strategy:
             matrix:
-                php: [ "8.0", "8.1", "8.2" ]
+                php: [ "8.0", "8.1", "8.2", "8.3" ]
```

Comment:
WordPress VIP does not support PHP 8.0 anymore. Min version in their background is PHP 8.1 for configuration.

Reply:
They don't support for new sites, but existing sites might still bee on 8.0. We have a few sites still on 8.0 right now.